### DISCUSSION TITLE
Github Blog

### DISCUSSION THREAD

**Discussion by bmiit145** [community/community#147272] [Other Feature Feedback, Questions, & Ideas] [answered]
### Body

How can I start to write a blog and contribute to github official blog website

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by tuves** [comment, marked as answer]
Hi @bmiit145,

The official Github Blog is maintained by articles written by GitHug staff. However, we also have the [readme](https://github.com/readme) project that often features developers. You can [nominate](https://github.com/readme/nominate) yourself if you have an interesting story! :)

**Reply by bmiit145** [reply]
Thank you :)

What we need to do in readme project
